.gitignore

```diff
@@ -28,3 +28,5 @@ synopsys-detect-*-help.json
 docker-test/
 *.hprof
 *.bak
+*.gradle
+LICENSE
```

Comment:
LICENSE is a source-code-controlled file updated when necessary by the build, so should not be ignored (at least not by developers).

Reply:
For sure, that popped up as a change in my desktop from the local build process, and since I wasn't intending to add it... but I'll unignore. ; )